UpdateProfile/handler.ts

```diff
@@ -103,11 +103,27 @@ export function UpdateProfileHandler(
       emailChanged ? false : existingProfile.isEmailValidated
     );
 
+    const autoEnableInboxAndWebHook =
+      existingProfile.acceptedTosVersion === undefined &&
+      profile.acceptedTosVersion !== undefined;
+
     const errorOrMaybeUpdatedProfile = await profileModel
       .update({
         ...existingProfile,
         // Remove undefined values to avoid overriding already existing profile properties
-        ...withoutUndefinedValues(profile)
+        ...withoutUndefinedValues(profile),
+
+        // Bugfix missing enable of Inbox and Webhook https://www.pivotaltracker.com/story/show/175095963
+        isInboxEnabled:
+          autoEnableInboxAndWebHook ||
+          fromNullable(profile.isInboxEnabled).getOrElse(
+            existingProfile.isInboxEnabled
+          ),
+        isWebhookEnabled:
+          autoEnableInboxAndWebHook ||
+          fromNullable(profile.isWebhookEnabled).getOrElse(
+            existingProfile.isWebhookEnabled
+          )
```

Comment:
You can also define something like

```
const overriddenInboxAndWebhook = autoEnableInboxAndWebHook ? {isInboxEnabled: true, isWebhookEnabled: true} : {}
```

and here use

```
...withoutUndefinedValues(profile),
...overriddenInboxAndWebhook
```

Reply:
Fixed in 4f6d4ea

Reply:
Fixed in 4f6d4ea